### DISCUSSION TITLE
Free worlds mission triggers - stuck halfway through campaign

### DISCUSSION THREAD

**Discussion by Vemenous-Repentile** [endless-sky/endless-sky#10398] [Q&A]
Hi - I'm doing another playthrough and I seem to be stuck halfway through the free worlds campaign. 

I'm at the stage where Alphecca, Alioth etc have joined the FW but I've not had offers of any new missions since. I rejected the offer to join the wolfpack earlier in the campaign but aside from that I've accepted and completed every free worlds missions I've been offered (as far as I know). 

Is there an easy fix to trigger the next mission, either by doing something in-game or via a quick save-file edit. 

Apologies if this has been asked and answered elsewhere. 

Thanks,
VR

**Comment by warp-core** [comment]
There's a short break in the story around this stage. Can you upload your save file so we can see where you are at?

**Reply by Vemenous-Repentile** [reply]
I can - thanks for the help!

[Electron Jones~3023-05-30 - Free worlds mission block.txt](https://github.com/user-attachments/files/16576362/Electron.Jones.3023-05-30.-.Free.worlds.mission.block.txt)

**Reply by warp-core** [reply]
It looks like you failed the mission "FW Albatross 2B" (named "Return to Dancer" in game).
You can edit your save file to be able to continue the mmissions.

Either remove the lines:
```
	"FW Albatross 2B: failed"
	"FW Albatross 2B: offered"
```
And return to Albatross, or change `failed` to `done` and land on Dancer.

**Reply by Vemenous-Repentile** [reply]
Much obliged - would've never found that by myself!

**Reply by Vemenous-Repentile** [reply]
[Electron Jones~3023-07-03 - xxx.txt](https://github.com/user-attachments/files/16576478/Electron.Jones.3023-07-03.-.xxx.txt)

Sorry - me again - I was able to follow up on the mission by making the edit you very kindly provided but upon landing on New Tibet the mission marker disappeared but none of the text popped up and the next mission didn't get flagged. 

Thanks!

**Reply by TheGiraffe3** [reply]
Can you take off and land again?

**Reply by Vemenous-Repentile** [reply]
Got a notification that someone had asked if I was able to take off and land again. That comment seemss to have disappeared, but the answer is yes - have tried a few times with no success
